internal/utils/gcs/gcs_utils.go

```diff
@@ -11,10 +11,11 @@ import (
 	speech "cloud.google.com/go/speech/apiv1"
 	"cloud.google.com/go/speech/apiv1/speechpb"
 	"cloud.google.com/go/storage"
+	"github.com/mmatongo/chew/internal/common"
 	"google.golang.org/api/option"
 )
 
-func uploadToGCS(ctx context.Context, client *storage.Client, bucket, filename string) (string, error) {
+func UploadToGCS(ctx context.Context, client *storage.Client, bucket, filename string) (string, error) {
```

Comment:
```suggestion
func Upload(ctx context.Context, client *storage.Client, bucket, filename string) (string, error) {
```

You are already in gcs package now